src/tools/add-timestamp-for-log.ts

```diff
@@ -0,0 +1,18 @@
+export function addTimestampForLog(log: string, arg?: any) {
+  let timestamp = changeTimeFormat();
+  if(arg) {
+	  console.log(timestamp, log, arg);
+  } else {
+	  console.log(timestamp, log);
+  }
+}
+
+function changeTimeFormat() {
```

Comment:
var d = new Date();
d.toLocaleString();       // -> "2/1/2013 7:37:08 AM"

No need to write it by ourself.